### DISCUSSION TITLE
[sos]单机服务不小心挂掉重启后collection.search会卡住不返回

### DISCUSSION THREAD

**Discussion by chuzhifeng** [milvus-io/milvus#16954] [Q&A and General discussion]
单机机器不小心磁盘占满了，然后机器重启后，重启milvus(v2.0.2)服务后，使用python检测服务时，在collection.search会卡死不返回了，请问是什么原因呢

**Comment by yhmo** [comment]
这种问题不太容易诊断，可以开一个issue，把重启后search卡住产生的日子贴上，方便研发诊断。可能是元数据和segment数据不一致导致的。
